README.md

```diff
@@ -89,16 +89,15 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 
 NOTE: p2 Proxy does not fully support the following as of yet:
 
-* Old style (non-p2) update sites. The code that accomplished this can be found [here](https://github.com/sonatype/nexus-public/blob/nexus-2.x/plugins/p2/nexus-p2-repository-plugin/src/main/java/org/sonatype/nexus/plugins/p2/repository/proxy/P2ProxyRepositoryImpl.java)
-  * These are fairly old, and have not been added at all
+* Old style (non-p2) update sites. These are fairly old, and have not been added at all.
```

Comment:
Trivial: I still don't see the value of the second sentence.  Old is said in the first sentence, have not been added at all I believe is inferred by the lead (does not fully support).  I guess technically "fully support" could infer this item is partially supported but why not just remove the word "fully" (with that second sentence).  Save yourself 12 words=)